agent.js

```diff
@@ -1063,7 +1066,8 @@ emitter.on(events.interactive, (data) => {
     errorLimit = 100;
     run(thisFile);
   } else if (thisCommand == "init") {
-    init();
+    await init();
+    process.exit(0);
```

Comment:
Doing a `process.exit()` to force the command to exit usually isn't right. Does our cli really not just exit?

Reply:
ya it doesn't exit. 
here's the [slack msg](https://testdriverai.slack.com/archives/C02M2V66NSC/p1730957153494809?thread_ts=1730957126.451829&cid=C02M2V66NSC) where I mentioned this

Reply:
it used to terminate before in `4.0` but weirdly it doesn't now

Reply:
maybe cause it spawns electron

Reply:
maybe, I tried `TD_OVERLAY=false` still it didn't terminate